### PR TITLE
Improve contributing guidelines

### PR DESCRIPTION
**Pull request by AnSavvides**
First of all thanks for being so accepting and welcoming to contributors outside of the core team that works on Radium. I really enjoy helping out & getting involved - and you lot have been great with considering every PR, giving feedback & being thankful.

I thought I would make a few changes to `CONTRIBUTING.md` to make it even easier & more straight-forward for someone to contribute to this project and set expectations (e.g. ensuring that PRs come with tests).

Any and all feedback is obviously welcome!

### CODE REVIEW

**Comment by alexlande** [comment]
This looks excellent. Thanks @AnSavvides :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,24 +2,38 @@ Thanks for contributing!
...
 
-Run `npm run lint` before committing to lint files.
+You will find all building blocks that make up Radium in the [`modules`](modules) folder.
 
-## Flow
+### Testing
 
-1. [Install flow](http://flowtype.org/docs/getting-started.html#installing-flow)
-2. Run `flow` to check for missing annotations and static type check failures
+You will find tests for each module inside [`modules/__tests__`](modules/__tests__). Whenever making any changes, ensure that all existing tests pass by running `karma start`.
```
Comment (ianobermiller) [inline CONTRIBUTING.md]: `npm test-dev` instead of `karma start`? Since `npm install` will not install global karma.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -2,24 +2,38 @@ Thanks for contributing!
...
+Before commiting any changes, be sure to do `npm run lint`; this will lint all relevant files using [ESLint](http://eslint.org/) and report on any changes that you need to make.
+
+### Flow
+
+- [Install flow](http://flowtype.org/docs/getting-started.html#installing-flow)
+- Run `flow` to check for missing annotations and static type check failures
+
+### Examples
+
+You can run examples locally by simply doing `npm run examples`; this will start a webpack dev server (the config file is found at [examples/webpack.config.js](examples/webpack.config.js)) making Radium examples available at `http://localhost:8080`.
```
Comment (ianobermiller) [inline CONTRIBUTING.md]: Note that if you use `npm run test-dev` as above, karma will use port 8080. You can change the port used by the examples by running `npm run examples -- --port 8000`.

File: CONTRIBUTING.md (hunk 1 above)
Comment (ryan-roemer) [inline CONTRIBUTING.md]: @ianobermiller Good catch.

**Comment by ianobermiller** [comment]
Very nice, thanks for cleaning this up. Do you think we should add something more formulaic? Something along the lines of:

Ready to submit a PR? Before you do, make sure to:
1. Run the test (you did add tests, right?) using `npm run test-dev`.
2. Run lint using `npm run lint`
3. Run flow using `flow`

File: CONTRIBUTING.md (hunk 1 above)
Comment (ianobermiller) [inline CONTRIBUTING.md]: Also note that Karma will keep running and run your tests everytime you make a change.

Comment (AnSavvides) [inline CONTRIBUTING.md]: Good call :+1:

File: CONTRIBUTING.md (hunk 2 above)
Comment (AnSavvides) [inline CONTRIBUTING.md]: :+1:

**Comment by AnSavvides** [comment]
Thanks! Great idea, I can add another sub-section called `Checklist before opening a PR`, or something like that, and add something along the lines of what you just suggested :)

**Pull request merged**